DependencyInjection/Configuration.php

```diff
@@ -47,6 +47,8 @@ public function getConfigTreeBuilder()
                     ->useAttributeAsKey('name')
                     ->prototype('array')
                     ->children()
+                        ->scalarNode('events_limit_per_run')
+                            ->defaultValue(200)->end()
```

Comment:
I think the default behavior should be to process all events in queue, shouldn't it ?